uber/configspec.ini

```diff
@@ -86,7 +86,7 @@ wristbands_enabled = boolean(default=False)
 # numbers have to be assigned in advance for those badge types, and new ones
 # cannot be created after those badges have been ordered.  Make this an empty
 # list to turn off these preassigned and customized badges.
-preassigned_badge_types = string_list(default=list('staff_badge', 'supporter_badge'))
+preassigned_badge_types = string_list(default=list())
```

Comment:
Yes - the system does not break if this is empty, and we want it empty by
default lest it cause unexpected results.

On Thu, Jun 16, 2016, 18:08 Dominic Cerquetti <notifications@github.com>
wrote:

> In uber/configspec.ini
> <https://github.com/magfest/ubersystem/pull/1867#discussion_r67433266>:
>
> > @@ -86,7 +86,7 @@ wristbands_enabled = boolean(default=False)
> >  # numbers have to be assigned in advance for those badge types, and new ones
> >  # cannot be created after those badges have been ordered.  Make this an empty
> >  # list to turn off these preassigned and customized badges.
> > -preassigned_badge_types = string_list(default=list('staff_badge', 'supporter_badge'))
> > +preassigned_badge_types = string_list(default=list())
>
> is this legit?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/magfest/ubersystem/pull/1867/files/71dcb9f651633dbf52eccc3d813daac1912356d2#r67433266>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AG3WB2oNZAi5IzW7bJZvsG4Ds1LqSRL-ks5qMcl6gaJpZM4I3erS>
> .
>